src/script/components/calling/Pagination.tsx

```diff
@@ -42,9 +42,12 @@ const Pagination: React.FC<PaginationProps> = ({totalPages, currentPage, onChang
         justifyContent: 'center',
         ...wrapperStyles,
       }}
+      data-uie-name="pagination-wrapper"
     >
       {pages.map(page => (
         <div
+          data-uie-name="pagination-item"
+          data-uie-active={currentPage === page ? 'active' : 'inactive'}
```

Comment:
Usually we use `data-uie-status` in such situations

Reply:
https://github.com/wireapp/wire-webapp/pull/11531/commits/1c60a8dbf03d37f5f77541ebf7de071ac7033cfe